src/main/java/roomescape/repository/ReservationTimeRepository.java

```diff
@@ -0,0 +1,59 @@
+package roomescape.repository;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.BeanPropertySqlParameterSource;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.ReservationTime;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+@Repository
+public class ReservationTimeRepository {
+
+    private final JdbcTemplate jdbcTemplate;
+    private final SimpleJdbcInsert jdbcInsert;
+
+    private final RowMapper<ReservationTime> timeRowMapper = (resultSet, rowNum) -> {
```

Comment:
RowMapper 사용 👍 조회 결과를 객체로 매핑할 때 재사용할 수 있겠군요~

상수로 사용해도 좋습니다

```suggestion
private static final RowMapper<ReservationTime> ROW_MAPPER
```